applications/luci-app-socatg/root/etc/uci-defaults/luci-socatg

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
If we use the socat backend from the package feed, then we do not need this, as it is reload/restart automatically by openwrt